composer.json

```diff
@@ -3,11 +3,10 @@
         "ext-curl": "*"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.5.1"
+        "phpunit/phpunit": "~4.5,>=4.5.1"
```

Comment:
Yeah @Danack, the caret operator has the same result if you use it as ```^4.5.1```. The only reason I usually suggest the tilde operator is because some people yet use older composer versions (where caret isn't available).